db/migrate/20180805120951_create_events.rb

```diff
@@ -0,0 +1,14 @@
+class CreateEvents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :events do |t|
+      t.string :title
+      t.string :place_title
+      t.string :place_cord
```

Comment:
This should not be a string.
We must take into account that place has **latitude** and **longitude**.
Consider creating two columns for each of the parameters with the following type.
```bash
:decimal, {:precision=>10, :scale=>7}
```

Note: you can find info in google about PostGIS extenstion, but it's overkill for us.

Reply:
@ZhenyaZhak I came to the conclusion that in this step we can remove place coordinates functionality at all.